README.md

```diff
@@ -129,9 +133,7 @@ Additional manual steps performed to collect population and or epidemiological d
 3. For city-level population data, the dataset is extracted from a compiled dataset of the 2010 Chinese City Statistical Yearbooks. We matched the city level population dataset to the city level COVID-19 epidemiology dataset. The file is in [data/raw/china/china_city_pop.csv](data/raw/china/china_city_pop.csv).
 
 ##### France
-1. Download the population data at the département level from the [National Institute of Statistics and Economic (INSEE) website](https://www.insee.fr/fr/statistiques/2012713#tableau-TCRD_004_tab1_departements) and save to [data/raw/france/TCRD_004.xls](data/raw/france/TCRD_004.xls).
-2. Download the crosswalk between département and région INSEE codes from the [National Institute of Statistics and Economic (INSEE) website](https://www.insee.fr/fr/information/3720946#titre-bloc-15) and save to [data/raw/france/departement2019.csv](data/raw/france/departement2019.csv).
```

Comment:
@bolliger32 is this deliberate? I can't see where it goes

Reply:
Yup, it's moved into an automated script (part of the adm info download and processing)